owl/SOMA-IO.owl

```diff
@@ -503,30 +501,32 @@ The Behaviour are the algoirthms used by the Software Instance, e.g., Bubble Sor
 
 The Structure is the organization of files, classes, objects and method calls of the program code."@en)
 SubClassOf(SOMA:Software <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design>)
+SubClassOf(SOMA:Software ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#describes> SOMA:Software_Configuration))
+SubClassOf(SOMA:Software ObjectAllValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#describes> ObjectIntersectionOf(SOMA:SoftwareInstance SOMA:Software_Configuration)))
```

Comment:
ObjectUnionOf perhaps here? It seems too harsh a request for something to be both a Configuration and an Agent ...

Reply:
What is the decision here? Can something even be both a Configuration and an Agent? (Both a software instance and a configuration)